### ISSUE TITLE
bigger dataset using material color palette

### ISSUE THREAD

**Issue by jusleg** [kcamcam/colorpicker#1]
[Google's material design](https://material-ui.com/style/color/#color-palette) already has this in their color palette. The color vs text color could be extracted and would form a large dataset

**Comment by kcamcam** [comment]
Good idea! The material-ui site's down right now. When it comes back up lets get us some more data

**Comment by almiche** [comment]
Toi meme. Thanks @jusleg

**Comment by kcamcam** [comment]
If ginger has no soul, is it even a color?

<sub>Sent with <a href="http://githawk.com">GitHawk</a></sub>
